pkg/proxy/iptables/proxier.go

```diff
@@ -417,6 +425,11 @@ func NewProxier(ipt utiliptables.Interface,
 		recorder:         recorder,
 		healthChecker:    healthChecker,
 		healthzServer:    healthzServer,
+		iptablesLines:    bytes.NewBuffer(nil),
```

Comment:
nit: This isn't really parsed into lines yet - call it `iptablesData` or `iptablesRaw` ?